openassessment/templates/openassessmentblock/edit/oa_edit_staff_assessment.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+{% load tz %}
+{% spaceless %}
+<li class="openassessment_assessment_module_settings_editor" id="oa_staff_assessment_editor">
+    <div class="drag-handle action"></div>
+    <div class="openassessment_inclusion_wrapper">
+        <input type="checkbox" id="include_staff_assessment"
+                {% if assessments.staff_assessment.required %} checked="true" {% endif %}>
+        <label for="include_staff_assessment">{% trans "Step: Staff Assessment" %}</label>
+    </div>
+    <div class="openassessment_assessment_module_editor">
+        <p id="staff_assessment_description" class="openassessment_description">
+            {% trans "Staff members assess learners' responses using the rubric for the assignment." %}
```

Comment:
@efischer19 In terms of XSS, the guideline is to escape everything, user data or not.  For example, in Mako templates, this is what will enable us to eventually switch to escaping by default.

This template seems to be a Django template, which is already safe by default, so I think all of these {% %} variables will be properly escaped.